core/vibes/soul/primitives/card-carousel/card-carousel.makeswift.tsx

```diff
@@ -0,0 +1,65 @@
+import { Image, Link, List, Select, Shape, Style, TextInput } from '@makeswift/runtime/controls';
+
+import { runtime } from '~/lib/makeswift/runtime';
+
+import { CardCarousel } from '.';
+
+interface Card {
+  title?: string;
+  imageSrc?: string;
+  imageAlt?: string;
+  link?: { href?: string; target?: string };
+}
+
+interface MSCardCarouselProps {
+  className: string;
+  textContrast: 'light' | 'dark';
+  cards: Card[];
+}
+
+runtime.registerComponent(
+  function MSCardCarousel({ className, textContrast, cards }: MSCardCarouselProps) {
+    return (
+      <CardCarousel
+        cards={cards.map(({ title, imageSrc, imageAlt, link }, index) => {
+          return {
+            id: title ?? index.toString(),
+            title: title ?? '',
+            image: { src: imageSrc ?? '', alt: imageAlt ?? '' },
+            href: link?.href ?? '',
+            textContrast,
+          };
+        })}
+        className={className}
+      />
+    );
+  },
+  {
+    type: 'primitive-card-carousel',
+    label: 'Primitives / Card Carousel',
+    icon: 'carousel',
+    props: {
+      className: Style(),
+      textContrast: Select({
+        options: [
+          { value: 'light', label: 'Light' },
+          { value: 'dark', label: 'Dark' },
+        ],
+        defaultValue: 'dark',
+      }),
+      cards: List({
```

Comment:
`List` as the name feels a bit generic. Thoughts on using `Slides`?

Reply:
Fixed